<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>addComputerDialog</class>
 <widget class="QDialog" name="addComputerDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>360</width>
    <height>214</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Add Computer</string>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="computerNameLabel">
     <property name="text">
      <string>Name:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="QLineEdit" name="computerNameLineEdit"/>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="computerYearLabel">
     <property name="text">
      <string>Year Made:</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QDateEdit" name="yearMadeEdit">
     <property name="font">
      <font>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="maximumDate">
      <date>
       <year>2016</year>
       <month>12</month>
       <day>13</day>
      </date>
     </property>
     <property name="displayFormat">
      <string>yyyy</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QComboBox" name="computerBuiltComboBox">
     <item>
      <property name="text">
       <string>Built</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Not built</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="computerTypeLabel">
     <property name="text">
      <string>Type:</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0" colspan="2">
    <widget class="QComboBox" name="computerTypeComboBox">
     <item>
      <property name="text">
       <string>Mechanical Computer</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Electronic Computer</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Transistors Machinery</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="QPushButton" name="addComputerButton">
     <property name="text">
      <string>Add</string>
     </property>
    </widget>
   </item>
   <item row="6" column="1">
    <widget class="QPushButton" name="cancelAddComputerButton">
     <property name="text">
      <string>Cancel</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>computerNameLineEdit</tabstop>
  <tabstop>yearMadeEdit</tabstop>
  <tabstop>computerBuiltComboBox</tabstop>
  <tabstop>computerTypeComboBox</tabstop>
  <tabstop>addComputerButton</tabstop>
  <tabstop>cancelAddComputerButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
